lms/static/js/search/app.js

```diff
@@ -0,0 +1,35 @@
+
+var edx = edx || {};
```

Comment:
If `edx` is undefined, won't this throw a `ReferenceError`? You're better off doing something like
```js
var edx = window.edx || {};
```